site/source/docs/tools_reference/emcc.rst

```diff
@@ -446,6 +446,7 @@ Options that are modified or new in *emcc* are listed below:
   The ``target`` file name extension defines the output type to be generated:
 
     - <name> **.js** : JavaScript (+ separate **<name>.wasm** file if emitting WebAssembly). (default)
+    - <name> **.mjs** : ES6 JavaScript module (+ separate **<name>.wasm** file if emitting WebAssembly).
```

Comment:
I think you also need to do `make text` in `site/`, which will update the `.txt` file that is printed from `emcc --help`.